split-webapp/split/src/components/TransactionList.js

```diff
@@ -67,6 +78,14 @@ class TransactionList extends React.Component {
         <ExpansionPanelDetails className="Payments">
           <div className="PaymentsTitle">
             <PaymentIcon className="PaymentHeaders" />
+            <div className="RunningTotal">
+              <div className="BillTitle Percentage">
+                ({Math.round((calculateTotalPaid(bill) / bill.total) * 100)}%){" "}
```

Comment:
I'm not sure this logic is actually quite correct because the payments won't add up to the total. The payments don't include the original payer's share which is included in the bill.total for now. This might also be wrong on the master branch right now, so we can think about this after the demo.